src/dom.js

```diff
@@ -112,6 +112,12 @@ export function documentContainsPolyfillInternal_(doc, element) {
  */
 export function removeElement(element) {
   if (element.parentElement) {
+    const activeElement = element.ownerDocument.activeElement;
+    // For accessibility reasons, restore focus to document if the focused
```

Comment:
What do browsers do by default? If this is good for a11y than then is it not the default?